Tests/SwiftDocCTests/Rendering/RenderNodeTranslatorSymbolVariantsTests.swift

```diff
@@ -500,72 +501,53 @@ class RenderNodeTranslatorSymbolVariantsTests: XCTestCase {
         )
     }
     
-    func testDictionaryKeysSectionVariants() throws {
-        func propertiesSection(in renderNode: RenderNode) throws -> PropertiesRenderSection {
-            let propertiesSectionIndex = 1
+    func testDictionaryKeysSection() throws {
+        let keySymbol = makeSymbol(id: "some-key", language: .data, kind: .dictionaryKey, pathComponents: ["SomeDictionary", "SomeKey"])
+        let catalog = Folder(name: "unit-test.docc", content: [
+            JSONFile(name: "ModuleName.symbols.json", content: makeSymbolGraph(moduleName: "ModuleName", symbols: [
+                makeSymbol(id: "some-dictionary", language: .data, kind: .dictionary, pathComponents: ["SomeDictionary"]),
+                keySymbol,
+            ]))
+        ])
+        
+        let (bundle, context) = try loadBundle(catalog: catalog)
+        let moduleReference = try XCTUnwrap(context.soleRootModuleReference)
+        let dictionaryReference = moduleReference.appendingPath("SomeDictionary")
+        
+        let node = try context.entity(with: dictionaryReference)
+        let symbol = try XCTUnwrap(node.semantic as? Symbol)
+        
+        func propertiesSection(for dictionaryKeysSection: DictionaryKeysSection) throws -> PropertiesRenderSection {
+            symbol.dictionaryKeysSection = dictionaryKeysSection
+            context.documentationCache[dictionaryReference] = node
             
-            guard renderNode.primaryContentSections.indices.contains(propertiesSectionIndex) else {
-                XCTFail("Missing properties section")
-                return PropertiesRenderSection(title: "Properties", items: [])
-            }
+            let converter = DocumentationNodeConverter(bundle: bundle, context: context)
+            let renderNode = try converter.convert(node)
```

Comment:
This generates a new warning: `No calls to throwing functions occur within 'try' expression`.

Looks like you avoided the existing test code in `assertMultiVariantArticle` ? And rewrote this test directly inline?

Reply:
That warning wasn't there in https://github.com/swiftlang/swift-docc/pull/1104/commits/4709dec537375c9be44acdc473eebf5e196fd5c1 😞 . It got introduced after merging `main`. Still, it's fixed in https://github.com/swiftlang/swift-docc/pull/1104/commits/276e7db5b9a66ac8395b68f203c60c163597d3c8